Dockerfile

```diff
@@ -61,11 +61,23 @@ RUN (cd /lib/systemd/system/sysinit.target.wants && for i in *; do [ $i == syste
 ## Remove any existing configurations
 RUN rm -f /etc/httpd/conf.d/*
 
+## Create the mount point for the authentication configuration files
+RUN mkdir /etc/httpd/auth-conf.d
+
+COPY docker-assets/entrypoint               /usr/bin
```

Comment:
Can we call this something more descriptive as it's no longer the entrypoint of the continer?

Reply:
I was thinking the same thing as I read it, and just thought I was wrong.